src/client/lcd/api/TxAPI.ts

```diff
@@ -371,7 +371,7 @@ export class TxAPI extends BaseAPI {
   ): Promise<Fee> {
     const gasPrices = options.gasPrices ?? this.lcd.config.gasPrices;
     const gasAdjustment =
-      options.gasAdjustment ?? this.lcd.config.gasAdjustment;
+      options.gasAdjustment ?? this.lcd.config.gasAdjustment ?? 5;
```

Comment:
The default value 5 is too big. I think 1.4 should be enough, but if you want to use it as an integer, use 2.

Reply:
Ok, it will use 2.